blueprints/sql-queries/blueprint.json

```
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"meta": {
+		"title": "Use runSql to run SQL queries",
+		"description": "Use runSql to run a SQL query that removes default posts from default theme and import a new set of posts from a WXR file",
+		"author": "juanmaguitar",
+		"categories": ["reset"]
+	},
+	"landingPage": "/?p=10",
+	"steps": [
+		{
+			"step": "login",
+			"username": "admin",
+			"password": "password"
+		},
+		{
+			"step": "resetData"
+		},
+		{
+			"step": "importWxr",
+			"file": {
+				"resource": "url",
+				"url": "https://raw.githubusercontent.com/wordpress-juanmaguitar/blueprints/trunk/blueprints/sql-queries/content.xml"
```

Would it be possible to use a link that points to `wordpress/blueprints` instead of your fork?


```suggestion
				"url": "https://raw.githubusercontent.com/wordpress/blueprints/trunk/blueprints/sql-queries/content.xml"
```

---

@bgrgicak that's fine, the automated job rewrites these urls upon merging. Linking to the fork in a PR is the only way to test the Blueprint

---

Thanks for the review and the clarification! I was struggling exactly with this.